demos/smartlab_demo/python/object_detection/settings.py

```diff
@@ -0,0 +1,185 @@
+        # self.random_size = (14, 26)
+        self.data_dir = None
+        self.train_ann = "instances_train2017.json"
+        self.val_ann = "instances_val2017.json"
+        # --------------- transform config ----------------- #
+        self.mosaic_prob = 1.0
+        self.mixup_prob = 1.0
+        self.hsv_prob = 1.0
+        self.flip_prob = 0.5
+        self.degrees = 10.0
+        self.translate = 0.1
+        self.mosaic_scale = (0.1, 2)
+        self.mixup_scale = (0.5, 1.5)
+        self.shear = 2.0
+        self.perspective = 0.0
+        self.enable_mixup = True
+        # --------------  training config --------------------- #
+        self.warmup_epochs = 5
+        self.max_epoch = 300
+        self.warmup_lr = 0
+        self.basic_lr_per_img = 0.01 / 64.0
+        self.scheduler = "yoloxwarmcos"
+        self.no_aug_epochs = 15
+        self.min_lr_ratio = 0.05
+        self.ema = True
+        self.weight_decay = 5e-4
+        self.momentum = 0.9
+        self.print_interval = 10
+        self.eval_interval = 10
+        self.exp_name = os.path.split(os.path.realpath(__file__))[1].split(".")[0]
+        # -----------------  testing config ------------------ #
+        self.test_size = (640, 640)
+        self.test_conf = 0.01
+        self.nmsthre = 0.65
+
+        self.depth = 0.33
+        self.width = 0.25
+        self.input_size = (416, 416)
+        self.mosaic_scale = (0.5, 1.5)
+        self.random_size = (10, 20)
+        self.test_size = (416, 416)
+        self.exp_name = os.path.split(os.path.realpath(__file__))[1].split(".")[0]
+        self.enable_mixup = False
+        # Define yourself dataset path
+        self.data_dir = ""
+        self.train_ann = ""
+        self.val_ann = ""
+        # define class map
+        self.num_classes = num_classes
+        if self.num_classes == 10:
+            self.mw_classes = mw_glb1cls10
+        elif self.num_classes == 6:
+            self.mw_classes = mw_glb2acls6
+        elif self.num_classes == 3:
+            self.mw_classes = mw_glb2bcls3
+        else:
+            raise ValueError
+        # define model file
+        if fp_model.endswith('.bin'):
+            self.fp_model = fp_model
+        # define dataset
+        if root_input is not None:
+            self.root_imgs = root_input
+            support_suffices = [".jpg", ".jpeg", ".webp", ".bmp", ".png"]
+            other_suffices = [s.upper() for s in support_suffices]
+            support_suffices += other_suffices
+            self.img_suffix = None
+            for f in os.listdir(self.root_imgs):
+                suffix = '.'+f.split('.')[-1]
+                if suffix in support_suffices:
+                    self.img_suffix = suffix
+                    break
+            assert self.img_suffix is not None
+        # define preview method
+        self.is_show = is_show
+        # define conditions
+        self.confthre = conf_thresh
+        self.nmsthre = nms_thresh
```

Comment:
Stay consistent in how you name var. You have different names for threshold throughout the demo

Reply:
fixed

Reply:
They are still different